src/pages/ProductList/ProductList.js

```diff
@@ -56,54 +58,23 @@ const ProductList = () => {
         </div>
         <div className="subMenu">
           <ul>
-            {SUB_MENU_LIST.map(submenu => {
-              return (
-                <li key={submenu.id}>
-                  <p>
-                    {submenu.menuName}
-                    <span>({submenu.quantity})</span>
-                  </p>
-                </li>
-              );
-            })}
-          </ul>
-        </div>
-        <div className="productList">
-          <ul>
-            <ProductListDetails />
+            {categoryList.sub_categories &&
+              categoryList.sub_categories.map(submenu => {
```

Comment:
옵셔널 체이닝을 활용하면 다음처럼 작성할 수 있습니다!
```suggestion
            {categoryList.sub_categories?.map(submenu => {
```

Reply:
넵~ 수정했습니다 :)